CONTRIBUTING.md

```diff
@@ -1,9 +1,9 @@
 # Contributing
 
-## Adding a module
+## Design
 
-Follow the instructions from [custom](/custom), place the module in the appropriate
-directory and document the module in `README.md`.
+Check out the "[Design Philosphy](./docs/explanation/design.md)" to ensure
+your changes are consistent.
```

Comment:
Almost feels like we're missing the last half of the sentence here but maybe just me? 
Personally, I'd reword it to always have the link be "last":

```suggestion
To understand our mindset in developing this plugin and how to ensure your changes align with that mindset, check out the "[Design Philosophy(./docs/explanation/design.md)."
```

What do you think?

(I'd personally appreciate it if the documentation generally followed that as it's also the same as GitHub's documentation style.)